Dockerfile

```diff
@@ -1,5 +1,46 @@
-FROM syndesis/caddy:v0.10.4
-MAINTAINER Syndesis Developers <syndesis@googlegroups.com>
+FROM docker.io/centos:7
 
-COPY Caddyfile /etc/Caddyfile
-COPY dist /srv
+ENV NGINX_VERSION 1.13.4-1.el7
+
+LABEL name="nginxinc/nginx" \
+      maintainer="Syndesis Authors <syndesis@googlegroups.com>" \
+      vendor="NGINX Inc." \
+      version="${NGINX_VERSION}" \
+      release="1" \
+      summary="NGINX" \
+      description="nginx will do ....." \
+### Required labels above - recommended below
+      url="https://www.nginx.com/" \
+      io.k8s.display-name="NGINX" \
+      io.openshift.expose-services="http:8080" \
+      io.openshift.tags="nginx,nginxinc"
+
+ADD nginx.repo /etc/yum.repos.d/nginx.repo
+
+RUN curl -sO http://nginx.org/keys/nginx_signing.key && \
+    rpm --import ./nginx_signing.key && \
+    yum -y install --setopt=tsflags=nodocs nginx-${NGINX_VERSION}.ngx && \
+    rm -f ./nginx_signing.key && \
+    yum clean all
+
+# forward request and error logs to docker log collector
+RUN ln -sf /dev/stdout /var/log/nginx/access.log && \
+    ln -sf /dev/stderr /var/log/nginx/error.log && \
+# Change pid file location & remove nginx user & change port to 8080
+    sed -i 's/\/var\/run\/nginx.pid/\/var\/cache\/nginx\/nginx.pid/g' /etc/nginx/nginx.conf && \
+    sed -i -e '/user/!b' -e '/nginx/!b' -e '/nginx/d' /etc/nginx/nginx.conf && \
+    rm -f /etc/nginx/conf.d/default.conf && \
+# modify perms for non-root runtime
+    chown -R 998 /var/cache/nginx /etc/nginx && \
+    chmod -R g=u /var/cache/nginx /etc/nginx
+
+#VOLUME ["/var/cache/nginx"]
+
+EXPOSE 8080 8443
```

Comment:
We don't yet, but now that OpenShift 3.6 properly supports re-encrypted routes to services using generated service serving cerificates, we will do ASAP (when our infra and OSO is at the appropriate release).